test/unit/mongoDBUtils.js

```diff
@@ -26,15 +26,16 @@ var MongoClient = require('mongodb').MongoClient,
     async = require('async');
 
 function cleanDb(host, name, callback) {
+
+    // FIXME: this code doesn't work with MongoDB replica set, we are assuming that an 
```

Comment:
Is this still todo? or will be done later?  Does the documentation need to be updated to reflect the current state (i.e. not working with replica sets)

Reply:
Just to clarify, as far as I understand (@dcalvoalonso could confirm) this IOTA works with replicat sets in the production code. What is not supported is the usage of replica set for unit tests.

Maybe it is worth a sentence about it in the "Testing" section of the documentation (the one which describes "npm test"... don't remember the .md file). Up to you @dcalvoalonso please.

Reply:
@fgalan is right. As it can be seen in https://github.com/telefonicaid/lightweightm2m-iotagent/pull/141#discussion_r232183287, this IOTA works with mongodb replica sets.

What it is not supported is running the unit tests against a mongodb replica set. We could fix this but to be honest I don't think that it would add any value...

I will follow @fgalan suggestion and I will add some text in the user guide document.

Reply:
Fixed with 0d83943a79fb61fa13d71ee3dc90a50a5d2430af